webpack_styles_lint.js

```diff
@@ -0,0 +1,45 @@
+var StyleLintPlugin = require("stylelint-webpack-plugin");
```

Comment:
we can remove this file now, right? since we're going with linting the raw files